plugin/sdk_v2/public/runtime/system_info.py

```
@@ -0,0 +1,123 @@
+                return cast(Result[JsonObject, SystemInfoImplError], Ok({"result": result}))
+            return cast(Result[JsonObject, SystemInfoImplError], Ok(result))
+        except AttributeError:
+            return Err(
+                CapabilityUnavailableError(
+                    "plugin_ctx.get_system_config is not available",
+                    op_name="system_info.get_system_config",
+                    capability="plugin_ctx.get_system_config",
+                    timeout=timeout,
+                )
+            )
+        except Exception as error:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)